docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -499,3 +504,14 @@ The output of the command will be similar to this:
 
   DATA: "b'\xa1fcCTdcH\x192B'"
   HEAD: "0c4364c6d5181282a1c7653038ec9515cb0530c6bfcb46f16e79b77cb524491676638339e8ff8e3cc57155c6d920e6a4d1f53947a31dc02908bcf68a91315ad5"
+
+
+Stopping Sawtooth Components
+============================
+
+To stop the validator, enter CTRL-c in the validator's terminal
+window. A single CTRL-c starts a graceful shutdown.
```

Comment:
I don't think we have any guarantees that the Validator will stop with just one CTRL-c. It was our intention that it would have a graceful shutdown, but other tasks have been more pressing.